src/components/GlobalHeader/index.tsx

```diff
@@ -28,7 +22,7 @@ export default function GlobalHeader (props: GlobalHeaderProps) {
     triggerResizeEvent()
   }
 
-  useEffect(() => triggerResizeEvent.cancel)
+  useEffect(() => triggerResizeEvent)
```

Comment:
The `cancel` was there because of the `Debounce` method.
The idea there was that when we unmount the component, we wanted to cancel the debounce event (but I don't know how it works ;) ).
So if we decide to remove `Debounce` decorator then we can remove this effect (whole line), otherwise we need to clean up and use this `cancel` method.

Reply:
the debounce method isnt loaded well into our setup and the compiler cant seem to find the cancel method. I attempted to drilldown but didnt get anywhere.

Reverting it now and will look into it with this issue https://trello.com/c/UyeKeGFI